.azure-pipelines/steps/run-snapshot-test.yml

```diff
@@ -128,3 +128,9 @@ steps:
     DD_LOGGER_DD_API_KEY: ${{ parameters.apiKey }}
   condition: succeededOrFailed()
   continueOnError: true
+
+- ${{ if eq(parameters.isLinux, true) }}:
```

Comment:
thx, I missed that when reading your changes :)

Reply:
I made the changes in the _other_ instance of Rider I had opened 🤦‍♂️